homeassistant/components/fujitsu_hvac/climate.py

```diff
@@ -0,0 +1,199 @@
+"""Support for Fujitsu HVAC devices that use the Ayla Iot platform."""
+from asyncio import timeout
+from contextlib import suppress
+from typing import Any
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+from ayla_iot_unofficial.fujitsu_hvac import Capability, FujitsuHVAC
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    API_TIMEOUT,
+    AYLA_APP_ID,
+    AYLA_APP_SECRET,
+    CONF_DEVICE,
+    CONF_EUROPE,
+    DOMAIN,
+    FAN_MODE_MAP,
+    HVAC_MODE_MAP,
+    SWING_MODE_MAP,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up one Fujitsu HVAC device."""
+    api = new_ayla_api(
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+        AYLA_APP_ID,
+        AYLA_APP_SECRET,
+        europe=entry.data[CONF_EUROPE],
+    )
+
+    try:
+        async with timeout(API_TIMEOUT):
+            await api.async_sign_in()
+    except (TimeoutError, AylaAuthError):
+        return
+
+    devices = await api.async_get_devices()
+    for dev in devices:
+        if dev.device_serial_number == entry.data[CONF_DEVICE]:
+            await dev.async_update()
+            async_add_entities([FujitsuHVACDevice(dev)])
+            return
+
+
+class FujitsuHVACDevice(ClimateEntity):
+    """Represent a Fujitsu HVAC device."""
+
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_precision = 0.5
```

Comment:
```suggestion
    _attr_precision = PRECISION_HALVES
```
there's a const for that